base/shell/cmd/lang/pt-BR.rc

```diff
@@ -0,0 +1,570 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN
```

Comment:
New file is added, but you have not included it in common RC file.

Reply:
Well, I'm a layman, but here's a try.